src/components/Avatar/Avatar.component.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
```

Comment:
I think the name of this component can be improved. `Avatar` sounds more like just an Icon with the profile picture /default image.